nethttp/server.go

```diff
@@ -88,12 +97,17 @@ func MiddlewareFunc(tr opentracing.Tracer, h http.HandlerFunc, options ...MWOpti
 		opNameFunc: func(r *http.Request) string {
 			return "HTTP " + r.Method
 		},
+		spanFilter:   func(r *http.Request) bool { return false },
```

Comment:
This is counter intuitive to me. I think it is more common to define Filter as a predicate that returns true for accepted entries, and false to not acceptable ones. This should also be explicitly stated in the comments to the new option function.

Reply:
@yurishkuro Good point. Just updated as suggested.